oop2-encapsulate-inherit.js

```diff
@@ -5,27 +5,269 @@
     - Publisher class.  It should have an authors and books property.
     - Review class.  It should have a rating and user property.
     -Prevent the properties from being accessed by outside code, using the encapsulation concept of "private variables".  Create methods that return the values of these private variables in case other pieces of the code need to access the data of the 4 classes.  Also create methods that allow outside code to "set" new values for the properties on the classes, without directly manipulating the values themselves. 
-
+*/
     //your code here...
```

Comment:
This file doesn't run. It highlights quite a few errors if you try to run it. Always run your code before submitting it.